src/Core/Checkout/Promotion/Subscriber/PromotionIndividualCodeRedeemer.php

```diff
@@ -34,10 +35,38 @@ public static function getSubscribedEvents(): array
     {
         return [
             CheckoutOrderPlacedEvent::class => 'onOrderPlaced',
+            AdminPromotionCodeRedeemedEvent::class => 'onCodeRedeemed',
```

Comment:
Why not point to the same function in the first place if you are calling the same function anyway?

Reply:
Had to tinker with the events, and then thought to keep it like this. but you are right, there're just extra lines that serve no purpose